lib/src/metta/runner/stdlib.metta

```diff
@@ -268,6 +268,10 @@
     (let $pattern $atom (let* $tail $template))
     $template )))
 
+(: merge-space (-> Atom %Undefined% Atom))
```

Comment:
`add-atom` returns `unit`, so this operation should also have `Unit` as its return type.

Reply:
And shouldn't the first argument be `Space`?